content/doc/pipeline/declarative/Advanced.adoc

```diff
@@ -0,0 +1,105 @@
+The pipeline you have seen so far has been mostly declarative. You can
```

Comment:
Thanks for that heads up, @bitwiseman. I had forgotten about the case issue. File name is as it was ported over from the wiki, but as @HRMPW pointed out, there is still some cut-and-polish to be done.